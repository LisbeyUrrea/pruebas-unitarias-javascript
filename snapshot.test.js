const multiSum = require('./public/index');
// const boton = require('./public/boton');

test('Ver si mi body cambi', () => {
    expect("<body> Hola mundo </body>").toMatchSnapshot()
});

// test('Generar boton', () => {
//     //arrage
//     const nombre = 'Enviar';
//     const clase = 'Boton';
//     const id = 'Boton';
//     const evento = (event) => alert("Hola Mundo");
//     //act
//     const result = boton.generar(nombre, clase, id, evento);
//     //Assert

//     expect(result).toMatchSnapshot()
// });

test('Ver si los estilos del body cambiaron', () => {
    expect("body { background-color: aqua; }").toMatchSnapshot()
});

test('hacer pruebas a un objeto javaScrip', () => {
    //arrage
    const numeroDos = [3, 4, 5, 6];
    //act
    const sumaNotas = multiSum(...numeroDos)
    const result = {
        sumaNotas: sumaNotas,
        nombre: 'Ana',
        clase: 'JavaScrip'
    }
    //Assert

    expect(JSON.stringify(result)).toMatchSnapshot()
});