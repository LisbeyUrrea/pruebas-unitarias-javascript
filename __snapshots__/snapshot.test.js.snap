// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ver si los estilos del body cambiaron 1`] = `"body { background-color: aqua; }"`;

exports[`Ver si mi body cambi 1`] = `"<body> Hola mundo </body>"`;

exports[`hacer pruebas a un objeto javaScrip 1`] = `"{\\"sumaNotas\\":18,\\"nombre\\":\\"Ana\\",\\"clase\\":\\"JavaScrip\\"}"`;
